### ISSUE TITLE
full-upgrade(v12): upgrade of configuration is done on all servers

### ISSUE THREAD

**Issue by nqb** [inverse-inc/packetfence#7381]
**Describe the bug**
When doing a cluster upgrade using do-upgrade.sh process, upgrade of configuration (using migration scripts) is performed on each server. Then we ask to run a configuration synchronisation on node A and B:

https://www.packetfence.org/doc/12.0.0/PacketFence_Clustering_Guide.html#_configuration_synchronisation

I don't think it's necessary to perform this step if we upgraded configuration in the same way on each node.

**Comment by julsemaan** [comment]
I agree with this (not necessary). Given our upgrade scripts are idempotent (i.e. can run multiple times yielding the same result), then that should never cause issues.

Still, no point on doing it for nothing so we should address this in an upcoming release
